IFC4x3/Properties/r/RelativePosition_06BnzJB6b9zhMKCUAtPkfJ/Documentation.md

```diff
@@ -0,0 +1 @@
+Indication of whether the welded joint lies to the left or the right in the track edge direction
```

Comment:
What is the track edge direction? Would it be better to use alignment direction to refer to something hopefully present in the dataset?